modules/ossm-kiali-about.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-kiali-about_{context}"]
+= About Kiali
+:context: ossm-kiali-about
+
+// there is only 1 attribute for Kiali at this time
+{KialiProduct} lets you view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
```

Comment:
Nit - per the [ISG section on anthropomorphism](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism), we should typically avoid assigning human characteristics inanimate objects (in this case the Operator "letting" you do x, y, z) in order to have more of a user focus. And I think this section, where a product and its features are being introduced to users, would benefit from that user focus. Here's what that might look like:
```suggestion
You can use {KialiProduct} to view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
```